packages/components/src/Datalist/Datalist.component.js

```diff
@@ -129,6 +154,10 @@ function Datalist(props) {
 		titleMap: props.titleMap,
 		filterValue,
 		multiSection: props.multiSection,
+		allowAddNewElements: props.allowAddNewElements,
+		allowAddNewElementsSuffix: props.allowAddNewElementsSuffix
+			? props.allowAddNewElementsSuffix
+			: t('NEW', { defaultValue: '(new)' }),
```

Comment:
```suggestion
		allowAddNewElementsSuffix: props.allowAddNewElementsSuffix ?? `(${ t('NEW', 'new' ) })`,
```
- I've used `??` operator
- I've replaced suffix by wrapping "new" translation with parenthesis

Reply:
Agree on the ?? operator
But for the parentheses, i would tend to say that it may depends of the locale if we want maybe some spacing or something

Reply:
Then IMO `NEW` i18n key is too generic to specify "new word with parenthesis"